pom.xml

```diff
@@ -26,9 +26,36 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.uber.tchannel</groupId>
-    <artifactId>tchannel</artifactId>
+    <artifactId>tchannel-java</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
```

Comment:
Its fine to update SNAPSHOT as long as people don't add it as the depedency, that is why a lot of people do multiple Alpha and Beta releases before releasing a final version.